scene/resources/importer_mesh.cpp

```diff
@@ -517,10 +517,10 @@ void ImporterMesh::generate_lods(float p_normal_merge_angle, float p_normal_spli
 					Vector3 normal = n0 * w + n1 * u + n2 * v;
 
 					Vector2 orig_uv = ray_uvs[j];
-					float orig_bary[3] = { 1.0f - orig_uv.x - orig_uv.y, orig_uv.x, orig_uv.y };
+					real_t orig_bary[3] = { 1.0f - orig_uv.x - orig_uv.y, orig_uv.x, orig_uv.y };
```

Comment:
Should the `1.0f` be changed?

Reply:
It doesn't need to, it has the same behavior as `(real_t)1.0` or similar. `float - double` will implicitly make the float become double, and `1.0f` is represented exactly.